### ISSUE TITLE
Radar Chart label color accessibility

### ISSUE THREAD

**Issue by niall-forrest** [apache/superset#27234]
## Screenshot
<img width="612" alt="Screenshot 2024-02-23 at 17 04 52" src="https://github.com/apache/superset/assets/86778302/6334f7d3-99da-485e-b6d1-ddcacf5bc6dc">

<img width="805" alt="image" src="https://github.com/apache/superset/assets/86778302/590ee5d8-83ae-402a-bcca-1de94edb8f19">

## Description

The Label colour in the Radar Chart is not very legible on the default background, and fails color accessibility standards.

## Design input

I would suggest aligning the Radar Chart label color with the other chart types. Aware there is ongoing work around theming which I hope will address label customization, however given this is an accessibility issue it may be a nice quick win. 

design:suggest
design:accessibility

**Comment by niall-forrest** [comment]
Yep no worries, I'll look into it and open a PR

**Comment by rusackas** [comment]
Closing this as stale since it's been silent for a couple hundred days again. Still happy to review a PR that addresses this, or revisit as necessary.
